src/value.rs

```diff
@@ -11,8 +11,27 @@ pub enum Value {
     Str(String),
     Object(HashMap<String, Value>),
     Array(Vec<Value>),
+    Bool(bool)
 }
 
+impl Value {
+    pub fn str(val: &str) -> Value {
```

Comment:
It's common to _me_, at least. I find having to write `to_owned()` for every single string literal to be really, _really_ annoying.  I personally find it clutters up the call site, making it harder to read without adding any extra useful information, so I usually use some sort of function to get rid of it wherever I can. 

Other people's opinions may vary, of course. But that's my 2c.